### DISCUSSION TITLE
Is there a multi tenant option?

### DISCUSSION THREAD

**Discussion by derekgardiner** [Budibase/budibase#6528] [Help] [answered]
Is there a way you can control crud rules around data captured via screens Budibase so that:

- An admin can see all data
- A client can only see the data they have created 
- A client can create users that only see the data belonging to that client

I can't seem to find anything in the documentation.

**Comment by gerhardmpl** [comment, marked as answer]
As far as I understood [this discussion](https://github.com/Budibase/budibase/discussions/3866#discussioncomment-2498572), there is no Multi-tenancy option for a single app.
